editor/src/components/inspector/inspector-common.ts

```diff
@@ -1401,3 +1412,122 @@ export function getConstraintsIncludingImplicitForElement(
 export function isHuggingParent(element: ElementInstanceMetadata, property: 'width' | 'height') {
   return element.specialSizeMeasurements.computedHugProperty[property] != null
 }
+
+function getGridElementBounds(
+  cell: ElementInstanceMetadata,
+  canvasContext: { scale: number; offset: CanvasVector },
+):
+  | null
+  | { type: 'contained'; gridRow: number; gridColumn: number }
+  | {
+      type: 'span'
+      gridRowStart: number
+      gridRowEnd: number
+      gridColumnStart: number
+      gridColumnEnd: number
+    } {
```

Comment:
Please don't do this in the return type. We should just create a type for each of these

Reply:
https://github.com/concrete-utopia/utopia/pull/6332/commits/78bed2c3dd550ae348c6d2a91195166a8080b6f1